src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/caseupdate/CaseUpdateClientTest.java

```diff
@@ -0,0 +1,263 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.client.caseupdate;
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A high number of imports can indicate a high degree of coupling within an object.](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4535703)